xdem/spatialstats.py

```diff
@@ -1067,6 +1298,179 @@ def variogram_sum(h, *args):
 
     return variogram_sum_fit, df_params
 
+def estimate_model_spatial_correlation(dvalues: Union[np.ndarray, RasterType], list_models: list[str | Callable],
+                                       estimator = 'dowd', gsd: float = None, coords: np.ndarray = None, subsample: int = 1000,
+                                       subsample_method: str = 'cdist_equidistant', n_variograms: int = 1,
+                                       n_jobs: int = 1, verbose = False,
+                                       random_state: None | np.random.RandomState | np.random.Generator | int = None,
+                                       bounds: list[tuple[float, float]] = None, p0: list[float] = None,
+                                       **kwargs) -> tuple[pd.DataFrame, pd.DataFrame, Callable[[np.ndarray], np.ndarray]]:
+
+    """
+    Estimate and model the spatial correlation of the input variable by empirical variogram sampling and fitting of a
+    sum of variogram model.
+
+    This function samples an empirical variogram using skgstat.Variogram, then optimizes by weighted least-squares the
+    sum of a defined number of models, using the functions `sample_empirical_variogram` and `fit_sum_model_variogram`.
+
+    :param dvalues: Proxy values of studied variable
```

Comment:
See my previous comment about the term "proxy values"

Reply:
See answer above.